packages/react/modules/hooks/use-adapter-subscription/use-adapter-subscription.ts

```diff
@@ -1,20 +1,34 @@
-import { TAdapter } from '@flopflip/types';
+import { TAdapter, TAdapterSubscriptionStatus } from '@flopflip/types';
 import React from 'react';
 
 function useAdapterSubscription(adapter: TAdapter) {
```

Comment:
Nit: `useGetAdapterSubscriptionStatus`?

Reply:
Yeah, wasn't sure myself. You don't have to use the getter and still something happens. The setup is more important (in Redux it's automatic). So I felt a more generic name is fine. But thanks for suggesting :)